src/views/Incorporations/DefineCompany.vue

```diff
@@ -63,7 +63,7 @@
 
     <section class="mt-10" v-show="isEntityType">
       <header id="office-address-header">
-        <h2>{{isTypeCoop ? 3 : 2 }}. Registered <span v-if="!isTypeCoop">and Records</span> Office
+        <h2>Registered <span v-if="!isTypeCoop">and Records</span> Office
```

Comment:
Great cleanup!

Reply:
Totally, this css fix you found works great. 
Makes a person wonder what other css tricks we could use 🤔